iree/samples/vulkan/BUILD

```diff
@@ -55,4 +56,7 @@ cc_binary(
         "@vulkan_headers//:vulkan_headers_no_prototypes",
         "@vulkan_sdk//:sdk",
     ],
+    linkopts = [
+        "-SUBSYSTEM:WINDOWS",
+    ],
```

Comment:
Need to test this on Linux to see if it's ignored or causes issues.

Ideas if needed:

* Add `select()` in Bazel
* Move into `iree_cc_binary()` for CMake

Reply:
Seems to be ignored by CMake. Linux/Bazel needed a select().